app/src/main/java/com/sunasterisk/fooddaily/data/model/FoodDetail.kt

```diff
@@ -55,4 +66,37 @@ data class FoodDetail(
             }
         }
     )
+
+    fun getRequiredTime(): String {
+        val timeRequired = StringBuilder().apply {
+            append(readyMinutes)
+            append(REQUIRED_TIME_UNIT)
+        }
+        return timeRequired.toString()
+    }
+
+    fun getPriceEstimate(): String {
+        val priceEstimate = StringBuilder().apply {
+            append(price)
+            append(PRICE_ESTIMATE_UNIT)
+        }
+        return priceEstimate.toString()
+    }
+
+    fun getIngredient(): String {
+        val value = StringBuilder()
+        val ingredients = ingredients
+        if (ingredients != null) {
+            for (ingredient in ingredients) {
+                value.apply {
+                    append(CHARACTER_MINUS)
+                    append(ingredient.original)
+                    append(CHARACTER_ENTER)
+                }
+            }
+        } else {
+            value.append(Resources.getSystem().getString(R.string.title_updating))
+        }
+        return value.toString()
```

Comment:
https://link.sun-asterisk.vn/CR3U3C
```suggestion
    fun getIngredient(): String? = ingredients?.map {
        CHARACTER_MINUS + it 
    }?.joinToString(CHARACTER_ENTER)

    // fragment 
    val ingredient = foodDetail.getIngredient() ?: getString(R.string.title_updating)
```